### PR TITLE
szavarce.py

### PR DESCRIPTION
**Pull request by szavarce42**
I have created a dictionary with uppercase letters index, which represent each morse code value. Then checked if there are two arguments to continue, otherwise will get out prompting an error message. Then the program will loop through the argument received (only the arg[1] since arg[0] is the name of the script). If the uppercase char is found in our dictionary, it will add it to a variable to display afterwards. If we find something different from the letters in the dictionary it will get out printing the error message. If we made it to the end, then we print the result stored in "resultado".

### CODE REVIEW

File: chall02/szavarce.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  print("usage: ./szavarce.py <a-zA-Z string>")
+  exit()
+
+for c in sys.argv[1]:
+  if c.upper() in codigo_morse:
+    resultado += codigo_morse[c.upper()]
+  else:
+    print("usage: ./szavarce.py <a-zA-Z string>")
+    exit()
+print("{}".format(resultado))
```
Comment (clafoutis42) [inline chall02/szavarce.py:17]: Why not just `print(resultado)`? :p

File: chall02/szavarce.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import sys
+codigo_morse = {
+  "A": ".-", "B": "-...", "C": "-.-.", "D": "-..", "E": ".", "F": "..-.", "G": "--.", "H": "....", "I": "..", "J": ".---", "K": "-.-", "L": ".-..", "M": "--", "N": "-.", "O": "---", "P": ".--.", "Q": "--.-", "R": ".-.", "S": "...", "T": "-", "U": "..-", "V": "...-", "W": ".--", "X": "-..-", "Y": "-.--", "Z": "--..", " ": " "
+}
+resultado = ""
+if len(sys.argv) != 2:
+  print("usage: ./szavarce.py <a-zA-Z string>")
```
Comment (clafoutis42) [inline chall02/szavarce.py:8]: The filename might change, you should make the message dynamic:
```suggestion
  print("usage: %s <a-zA-Z string>" % sys.argv[0])
```

File: chall02/szavarce.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  exit()
+
+for c in sys.argv[1]:
+  if c.upper() in codigo_morse:
+    resultado += codigo_morse[c.upper()]
+  else:
+    print("usage: ./szavarce.py <a-zA-Z string>")
+    exit()
```
Comment (clafoutis42) [inline chall02/szavarce.py:16]: You could have made this more pythonic with those oneliners:
```suggestion
if not sys.argv[1].replace(" ", "").isalpha():
    print("usage: ./szavarce.py <a-zA-Z string>")
    exit()
resultado = "".jon(codigo_morse[c] for c in sys.argv[1])
```

File: chall02/szavarce.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+resultado = ""
+if len(sys.argv) != 2:
+  print("usage: ./szavarce.py <a-zA-Z string>")
+  exit()
+
+for c in sys.argv[1]:
+  if c.upper() in codigo_morse:
+    resultado += codigo_morse[c.upper()]
+  else:
+    print("usage: ./szavarce.py <a-zA-Z string>")
```
Comment (clafoutis42) [inline chall02/szavarce.py:15]: You wrote this multiple times, maybe you should use a function to improve readability.

**Review by clafoutis42 — COMMENTED** [review]
Good job, you have a working solution, however it is not very pythonic. You are writing your code as if you were using C, I suggest you read [Fluent Python](https://www.amazon.com/Fluent-Python-Concise-Effective-Programming/dp/1491946008) if you want to keep progressing in python!

However as your file is not executable, I can't accept this PR :(

**Comment by szavarce42** [comment]
Hello, thanks for your comments. What do u mean it is not executable? I
could execute it in my pc... and you asked for a <login>.py file...

**Pull request closed without merging**